tests/common/plugins/conditional_mark/tests_mark_conditions_skip_traffic_test.yaml

```diff
@@ -116,3 +116,50 @@ everflow/test_everflow_testbed.py:
     reason: "Skip traffic test for KVM testbed"
     conditions:
       - "asic_type in ['vs']"
+
+#######################################
+#####            fib              #####
+#######################################
+fib/test_fib.py:
+  skip_traffic_test:
+    reason: "Skip traffic test for KVM testbed"
+    conditions:
+      - "asic_type in ['vs']"
+
+fib/test_fib.py::test_ipinip_hash:
+  skip_traffic_test:
+    reason: "Skip traffic test for KVM testbed"
+    conditions:
+      - "asic_type in ['vs']"
+
+#######################################
+#####            ip               #####
+#######################################
+ip/test_ip_packet.py:
+  skip_traffic_test:
+    reason: "Skip traffic test for KVM testbed"
+    conditions:
+      - "asic_type in ['vs']"
+
+# Temporary add because conditional mark would only match longest prefix
+ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_drop:
+  skip_traffic_test:
+    reason: "Skip traffic test for KVM testbed"
+    conditions:
+      - "asic_type in ['vs']"
+
+# Temporary add because conditional mark would only match longest prefix
+ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_tolerant:
+  skip_traffic_test:
+    reason: "Skip traffic test for KVM testbed"
+    conditions:
+      - "asic_type in ['vs']"
+
```

Comment:
I see in `line 138` the entire test module has been set for skip. Is there a reason to choose these two tests specifically ? Won't they be skipped anyway as part of the module skip ?

Reply:
Currently conditional mark would only match longest prefix, `ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_drop` and `ip/test_ip_packet.py::TestIPPacket::test_forward_ip_packet_with_0xffff_chksum_tolerant` were defined in `tests\common\plugins\conditional_mark\tests_mark_conditions.yaml`, so the module skip won't be matched in these 2 testcases. Yutong will fix it, but we need a temporary workaround